.travis.yml

```diff
@@ -72,12 +72,12 @@ jobs:
         - travis_retry composer require --dev --prefer-dist --prefer-stable doctrine/coding-standard:^4.0.0
       script: vendor/bin/phpcs
 
-    - stage: Static Analysis
-      php: 7.1
-      env: DEPENDENCIES=""
-      before_script:
-        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9.2
-      script: vendor/bin/phpstan analyse -l 5 -c phpstan.neon src
+    #- stage: Static Analysis
+    #  php: 7.1
+    #  env: DEPENDENCIES=""
+    #  before_script:
+    #    - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9.2
+    #  script: vendor/bin/phpstan analyse -l 5 -c phpstan.neon src
```

Comment:
I suppose it is impossible to analyse this codebase due to autoloading reflection?

Reply:
Yes, exactly.